DeKast/src/app/register/register.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators, AbstractControl, ValidatorFn   } from '@angular/forms'
+import { ToastService } from '../toast.service';
+
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.scss']
+})
+export class RegisterComponent implements OnInit {
+
+  registrationForm = new FormGroup({
+    firstName: new FormControl('', Validators.required),
+    lastName: new FormControl('', Validators.required),
+    email: new FormControl('', [Validators.required, Validators.email]),
+    phoneNumber: new FormControl(''),
+    password: new FormControl('', [Validators.required, Validators.minLength(5)]),
+    repeatPassword: new FormControl('')
+  });
+
+  constructor(private toastService: ToastService) { }
+
+  ngOnInit() {
+  }
+
+  callingFunction() {
+    if (this.registrationForm.valid) {
```

Comment:
haal de console logs eruit en de alert shit. Kun je proberen Register met Firebase te doen? Alleen email and password moet, dus je hoeft geen collection te maken